package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lemmy-js-client",
   "description": "A javascript / typescript client for Lemmy",
-  "version": "0.11.3-rc.4",
+  "version": "0.11.4-rc.8",
```

Comment:
@shilangyu I'd like you to look over these API changes, some of them are breaking, but they made sense when adding person and community blocks.